app.py

```diff
@@ -70,17 +70,16 @@ def education(index):
     '''
     Handles education requests
     '''  
-    if request.method == 'GET' and index == "1":
-        return jsonify(data["education"][0])
-    elif request.method == 'GET' and index == "2":        
-        return jsonify(data["education"][1]) 
-    elif request.method == 'GET' and index == "3":        
-        return jsonify(data["education"][2])        
+    if request.method == 'GET' and index.isnumeric():
+        index_num = int(index)
```

Comment:
@SuelenKarbivnychyy 

great work! thanks for incorporating my suggestion.

one question: why do you have to create `index_num` if the `index` variable is already numeric (and you know that because you do `index.isnumeric()`)?

Reply:
Thank you, 

The **index.isnumeric()** is just checking if the index itself is the numeric type, it's not converting the variable **index** to be the numeric. I handled this way because I also want to handle the edge case of the index being string, in that case, it will also return an Error message.
I just added the code for this last part.

Reply:
Great! Thanks for the explanation.